package.json

```diff
@@ -4,9 +4,6 @@
     "url": "http://github.com/broofa",
     "email": "robert@broofa.com"
   },
-  "engines": {
-    "node": ">=6.0.0"
```

Comment:
How about `"node": ">=4.0.0"`?  I'd rather be explicit about what this is known to work on.